src/Plugin/EntityBrowser/WidgetSelector/Tabs.php

```diff
@@ -25,31 +25,20 @@ class Tabs extends WidgetSelectorBase {
    * {@inheritdoc}
    */
   public function getForm(array &$form = array(), FormStateInterface &$form_state = NULL) {
-    // Set a wrapper container for us to replace the form on ajax call.
-    $form['#prefix'] = '<div id="entity-browser-form">';
-    $form['#suffix'] = '</div>';
-
+    $element = [];
     foreach ($this->widget_ids as $id => $label) {
-      $element[$label] = array(
+      $name = 'tab_selector_' . $id;
+      $element[$name] = array(
         '#type' => 'button',
         '#value' => $label,
         '#disabled' => $id == $this->getDefaultWidget(),
         '#executes_submit_callback' => TRUE,
-        '#limit_validation_errors' => array(array($label)),
-        '#ajax' => array(
-          'callback' => array($this, 'changeWidgetCallback'),
-          'wrapper' => 'entity-browser-form',
-        ),
+        '#limit_validation_errors' => array(array($id)),
+        '#name' => $name,
+        '#widget_id' => $id,
       );
     }
 
-    $element['change'] = array(
```

Comment:
Not needed at all. I suspect this code was copied from dropdown widget which uses this element.